File: android_mobile/src/main/java/com/alexstyl/specialdates/people/ImportFromFacebookViewHolder.java
package com.alexstyl.specialdates.people;

import android.support.v7.widget.RecyclerView;
import android.view.View;

class ImportFromFacebookViewHolder extends RecyclerView.ViewHolder {

    ImportFromFacebookViewHolder(View rowView) {
        super(rowView);
    }

    public void bind(final PeopleViewHolderListener listener) {
     itemView.setOnClickListener(new View.OnClickListener() {
         @Override
         public void onClick(View view) {
             listener.onFacebookImport();
         }
     });
    }
}

File: android_mobile/src/main/java/com/alexstyl/specialdates/people/ImportFromFacebookViewHolder.kt
package com.alexstyl.specialdates.people

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView

internal class ImportFromFacebookViewHolder(rowView: View, private val label: TextView) : RecyclerView.ViewHolder(rowView) {

    fun bind(viewModel: FacebookImportViewModel, listener: PeopleViewHolderListener) {
        label.text = viewModel.label
        itemView.setOnClickListener { listener.onFacebookImport() }
    }
}
